google-cloud-core/src/main/java/com/google/cloud/CivilDate.java

```diff
@@ -22,17 +22,28 @@
 import java.util.regex.Pattern;
 
 /**
- * Represents a Date without time, such as 2017-03-17. Date is timezone independent.
+ * Represents a date without time or timezone, e.g. 2017-03-17.
+ *
+ * CivilDate holds values for BigQuery DATE
+ * (https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#date-type)
+ * and Spanner DATE
+ * (https://cloud.google.com/spanner/docs/data-types#date-type).
+ *
+ * The DATE type represents a logical calendar date, independent of time zone. A DATE value
```

Comment:
Oh nvm I was confusing that the DATE should be replaced with CivilDate... but it seems like I misunderstood.